client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -452,6 +452,7 @@ export const downloadBatchTallySheets = async (
           right: pageMargin,
           top: pageMargin,
         },
+        pageBreak: 'avoid',
```

Comment:
Considering only adding this for contests 2+, as we probably want contest 1 printing on the first page always